executor/simple.go

```diff
@@ -1098,6 +1101,26 @@ func (e *SimpleExec) executeFlush(s *ast.FlushStmt) error {
 	return nil
 }
 
+func (e *SimpleExec) executeAlterInstance(s *ast.AlterInstanceStmt) error {
+	if s.ReloadTLS {
+		logutil.BgLogger().Info("execute reload tls")
```

Comment:
Not sure if it is necessary or not.

Reply:
maybe better keep this log, reload tls is critial operation, we should keep this log to help diagnose, we also add s.NoRollbackOnError value to this log in new commit